_data/abishanka/abishanka.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Abishanka Saha
+institution: DPS Sushant Lok
+profile_pic: abishanka.png
+quote: Shoot for the Moon and then maybe you'll have a chance at Mars {*_*}
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the special character to maintain the same format